docs/src/componentDocs/EmptyState/playground/PlaygroundPage.tsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import {
+    InputConfig,
+    PreviewComponent,
+    CodeSnippetFunction,
+    getPropsToString,
+    getPropsMapping,
+    Playground,
+} from '@brightlayer-ui/react-doc-components';
+import Stack from '@mui/material/Stack';
+import { EmptyState, EmptyStateProps } from '@brightlayer-ui/react-native-components';
+import Add from '@mui/icons-material/Add';
+import Button from '@mui/material/Button';
+import { IconSource } from '@brightlayer-ui/react-native-components/core/__types__';
+
+const inputConfig: InputConfig = [
+    // Required Props
+    {
+        id: 'title',
+        type: 'string',
+        typeLabel: 'ReactNode',
+        description: 'The primary text to display (first line)',
+        required: true,
+        initialValue: 'No devices',
+        category: 'Required Props',
+    },
+    // Optional Props
+    {
+        id: 'description',
+        type: 'string',
+        typeLabel: 'ReactNode',
+        description: 'The secondary text to display (second line)',
+        required: false,
+        initialValue: 'Check your network connection',
+        category: 'Optional Props',
+    },
+    {
+        id: 'icon',
+        type: 'select',
+        typeLabel: 'IconSource',
+        description: 'A component to render for the primary icon',
+        initialValue: 'devices',
+        options: [
+            { label: 'undefined', value: 'undefined' },
+            { label: 'devices', value: 'devices' },
+        ],
+        required: true,
+        category: 'Optional Props',
+    },
+    {
+        id: 'iconSize',
+        type: 'number',
+        typeLabel: `number`,
+        initialValue: 100,
+        description: 'The size of the primary icon',
+        required: false,
+        minValue: 100,
+        maxValue: 200,
+        valueStep: 4,
+        category: 'Optional Props',
+        defaultValue: 100,
+    },
+    {
+        id: 'iconColor',
+        label: 'iconColor',
+        type: 'color',
+        typeLabel: 'string',
+        description: 'The color of the primary icon',
+        required: false,
+        category: 'Optional Props',
+        defaultValue: 'inherit',
+        allowMuiColors: true,
+    },
+    // Other Configuration
+    {
+        id: 'showAction',
+        label: 'Show Actions',
+        type: 'boolean',
+        description: 'Whether to show action section or not',
```

Comment:
change description to this `Additional components to render below the text`

Reply:
Changed